raftpb/raft.proto

```diff
@@ -71,22 +75,32 @@ message Message {
 	optional uint64      from        = 3  [(gogoproto.nullable) = false];
 	optional uint64      term        = 4  [(gogoproto.nullable) = false];
 	// logTerm is generally used for appending Raft logs to followers. For example,
-	// (type=MsgApp,index=100,logTerm=5) means leader appends entries starting at
-	// index=101, and the term of entry at index 100 is 5.
+	// (type=MsgApp,index=100,logTerm=5) means the leader appends entries starting
+	// at index=101, and the term of the entry at index 100 is 5.
 	// (type=MsgAppResp,reject=true,index=100,logTerm=5) means follower rejects some
 	// entries from its leader as it already has an entry with term 5 at index 100.
+	// (type=MsgStorageAppendResp,index=100,logTerm=5) means the local node wrote
```

Comment:
Is this entirely correct? In `newStorageAppendRespMsg`, we're putting the lastIndex/lastTerm only conditionally. Also, it's the lastIndex/lastTerm, not that of the entries passed in.

Reply:
I think it is what we want to say though. The delivery of a `MsgStorageAppendResp` with `index=100,logTerm=5` means that entries up to index=100/term=5 have been written in stable storage. It doesn't necessarily mean that they were written by the corresponding `MsgStorageAppend` though. I added a sentence to call that out.